sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/TestUtils.java

```diff
@@ -37,27 +32,31 @@
  * Contains helper methods for generating inputs for test methods
  */
 final class TestUtils {
-    static final String URL_TEST_FILE_FORMAT = "https://raw.githubusercontent.com/Azure/azure-sdk-for-java/"
-        + "master/sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/sample_files/Test/";
+    // Duration
+    static final Duration ONE_NANO_DURATION = Duration.ofNanos(1);
```

Comment:
only used at one place, no need to define it in a constant.

Reply:
I just reorder them. Haven't check the existence.

Reply:
checked. It used in two places.